birdhouse/components/cowbird/docker-compose-extra.yml

```diff
@@ -65,6 +69,20 @@ services:
       - ./components/cowbird/config/cowbird/config.yml:/opt/local/src/cowbird/config/cowbird.yml
       - ./components/cowbird/config/cowbird/cowbird.ini:/opt/local/src/cowbird/config/cowbird.ini
       - ./components/cowbird/config/cowbird/celeryconfig.py:/opt/local/src/cowbird/config/celeryconfig.py
-      - "${DATA_PERSIST_ROOT}/${USER_WORKSPACES}:/${USER_WORKSPACES}"
+      - "${DATA_PERSIST_ROOT}/${USER_WORKSPACES}:${DATA_PERSIST_ROOT}/${USER_WORKSPACES}"
     restart: always
     logging: *default-logging
+
+  # Dedicated database for Cowbird, since other 'mongodb' image does not employ the same version.
+  cowbird-mongodb:
+    image: mongo:${COWBIRD_MONGODB_VERSION}
+    container_name: ${COWBIRD_MONGODB_SERVICE}
+    networks:
+      - "${COWBIRD_MONGODB_SERVICE}"
```

Comment:
Since the name is hardcoded in `networks` below, it is better to hardcode it here as well, otherwise they could mismatch if the variable is overridden.

Idem in `cowbird` and `cowbird-worker` services.